fb_instant_articles.module

```diff
@@ -18,6 +18,16 @@ function fb_instant_articles_menu() {
     'file path' => drupal_get_path('module', 'system'),
   );
 
+  $items['admin/config/services/fb-instant-articles/settings'] = array(
```

Comment:
@simonengelhardt interesting, I do see that :eyes: I just updated the config links though in ad92018, which seems better. I'm curious about the form issue you're seeing though. Maybe `drush cc menu`? (i assume you probably did that, but just in case).